mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -80,6 +81,7 @@ export class ExperimentPage extends Component {
     const orderBy = ExperimentPage.getOrderByExpr(orderByKey, orderByAsc);
     const viewType = lifecycleFilterToRunViewType(lifecycleFilter);
     this.setState({ loadingMore: true });
+    this.loadMoreRunsRequestId = getUUID();
```

Comment:
Previously, we'd incorrectly reuse the same UUIDs here across load more/search requests

Reply:
Why using the same UUID is considered incorrect? Using different UUID will end up with a large amount of entries in the **apis** of the redux store and eventually OOM the browser if user keep does not refresh for a long time.